### PR TITLE
BoxDecoration backgroundColor is now color

### PR DESCRIPTION
**Pull request by mravn-google**

### CODE REVIEW

File: _includes/_code/container/main.dart [hunk 1]
```diff
@@ -38,7 +38,7 @@ class _MyHomePageState extends State<MyHomePage> {
   Widget build(BuildContext context) {
     var container = new Container(
       decoration: new BoxDecoration(
-        backgroundColor: Colors.black26,
+        color: Colors.black26,
       ),
```
Comment (abarth) [inline _includes/_code/container/main.dart]: I'd just simplify this to:

`new Container(color: Colors.black26)`

**Comment by abarth** [comment]
LGTM

**Comment by sethladd** [comment]
I think we just merged this in https://github.com/flutter/website/pull/530

**Pull request closed without merging**